app/presenters/hyku/work_show_presenter.rb

```diff
@@ -21,8 +21,20 @@ def isbns
       isbns&.flatten&.compact
     end
 
+    def video_embed_viewer?
+      extract_video_embed_presence
+    end
+
+    def video_embed_viewer
+      :video_embed_viewer
+    end
+
     private
 
+      def extract_video_embed_presence
+        solr_document[:video_embed_tesim].present?
```

Comment:
Why not replace the contents of line 25 with this line, and then remove the `extract_video_embed_presence`

Reply:
I updated the code.